index.html

```diff
@@ -126,187 +846,604 @@ <h3>
             <dd>
               The requested receive buffer size, in bytes.
             </dd>
+            <dt>
+              <dfn>readableStreamBufferSize</dfn> member
+            </dt>
+            <dd>
+              The internal queue size for {{UDPSocket/[[readable]]}}.
+            </dd>
           </dl>
-          <div class="note">
-            The only current implementation (Chromium) ignores
-            {{SocketOptions/localAddress}} and {{SocketOptions/localPort}}.
-          </div>
         </section>
-        <section data-dfn-for="TCPSocketOptions">
-          <h3>
-            `TCPSocketOptions` dictionary
-          </h3>
+
+      </section>
+
+      <section>
+        <h3>{{UDPSocket/[[readable]]}} attribute (internal)</h3>
+
+        <aside class="example">
+          An application receiving data from a UDP socket will typically use the {{ReadableStream}} like this:
+
+          <pre class="js">
+            const { readable } = await socket.connection;
+
+            const reader = readable.getReader();
+
+            while (true) {
+              const { value, done } = await reader.read();
+              if (done) {
+                // |reader| has been canceled.
+                break;
+              }
+              const { data } = value;
+              // Do something with |data|...
+            }
+
+            reader.releaseLock();
+          </pre>
+        </aside>
+
+        The steps to <dfn data-lt="initialize udp readable stream">initialize the UDP readable stream</dfn> are:
+
+        <ol>
+          <li>Let |stream:ReadableStream| be a [=new=] {{ReadableStream}}.
+          <li>
+            Let |pullAlgorithm| be the following steps:
+            <ol>
+              <li>
+                Let |desiredSize| be the
+                <a href="https://streams.spec.whatwg.org/#desired-size-to-fill-a-streams-internal-queue">desired size</a>
+                of [=this=].{{UDPSocket/[[readable]]}}'s
+                <a href="https://streams.spec.whatwg.org/#internal-queues">internal queue</a>.
+              <li>
+                Run the following steps |desiredSize| times [=in parallel=].
```

Comment:
Rather than running the steps multiple times, describe how the Chromium implementation does it which is ask the OS for up to `desiredSize` packets and enqueue each as they are received.

Reply:
Looks good to me.